.github/CONTRIBUTING.md

```diff
@@ -1,152 +1,86 @@
 # Contributing to PennyLane
 
 Thank you for taking the time to contribute to PennyLane!
-:confetti_ball: :tada: :fireworks: :balloon:
 
-PennyLane is a collaborative effort with the quantum computation and machine learning
-community - while we will continue working on adding new and exciting features to PennyLane,
-we invite you to join us and suggest improvements, research ideas, or even just to discuss how
-PennyLane fits into your workflow.
+PennyLane is a collaborative effort with the quantum computation and machine learning communities. We invite you to join our team in whatever capacity works best for you.
```

Comment:
(Minor: would be good to add line-breaks at ~100 characters, to reduce the size of future PRs modifying this file)